### PR TITLE
GSYE-636: Reactivated HeatPumpEnergyParameters unit tests.

### PR DESCRIPTION
**Pull request by spyrostz**
## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=master
**GSY_FRAMEWORK_BRANCH**=master

### CODE REVIEW

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: tests/strategies/energy_parameters/test_heat_pump.py [hunk 1]
```diff
@@ -67,17 +89,19 @@ def test_event_traded_energy_updates_temp_increase(energy_params):
...
+        assert isclose(energy_params.get_min_energy_demand_kWh(CURRENT_MARKET_SLOT),
+                       0.8865112724493694)
 
     @staticmethod
     def test_get_max_energy_demand_kWh_returns_correct_value(energy_params):
         energy_params.event_market_cycle(CURRENT_MARKET_SLOT)
         energy_params.get_max_energy_demand_kWh(CURRENT_MARKET_SLOT)
         assert (energy_params.get_max_energy_demand_kWh(CURRENT_MARKET_SLOT) ==
-                0.7652439024390245)
+                3.0)
```
[the remark this answers is not in the document]
Comment (spyrostz) [reply tests/strategies/energy_parameters/test_heat_pump.py:107]: I think not, I think that these tests were last executed some time while developing the heatpump (their last history is from 2022, therefore I am fairly certain that other changes affected these too). All tests failed BTW when I first ran them, due to not passing any input parameters on the HeatpumpEnergyParameters fixture.

File: tests/strategies/energy_parameters/test_heat_pump.py [hunk 2]
```diff
@@ -57,6 +77,8 @@ def test_event_traded_energy_decrements_posted_energy(energy_params):
         original_min_demand = energy_params.state._min_energy_demand_kWh[CURRENT_MARKET_SLOT]
         original_max_demand = energy_params.state._max_energy_demand_kWh[CURRENT_MARKET_SLOT]
         energy_params.event_traded_energy(CURRENT_MARKET_SLOT, traded_energy)
+        print(original_max_demand)
```
Comment (hannesdiedrich) [inline tests/strategies/energy_parameters/test_heat_pump.py:80]: I will remove these in my PR

Comment (spyrostz) [reply tests/strategies/energy_parameters/test_heat_pump.py:80]: Thanks!

**Pull request merged**